probe-rs/src/config/target.rs

```diff
@@ -124,16 +125,31 @@ impl Target {
         } else if chip.name.starts_with("nRF9160") {
             tracing::warn!("Using custom sequence for nRF9160");
             debug_sequence = DebugSequence::Arm(Nrf9160::create());
-        } else if chip.name.starts_with("STM32H7") {
-            tracing::warn!("Using custom sequence for STM32H7");
-            debug_sequence = DebugSequence::Arm(Stm32h7::create());
+        } else if chip.name.starts_with("STM32F0") {
+            tracing::warn!("Using custom sequence for ARMv6 {}", chip.name);
+            debug_sequence = DebugSequence::Arm(Stm32Armv6::create(Stm32Armv6Family::F0));
+        } else if chip.name.starts_with("STM32L0") {
+            tracing::warn!("Using custom sequence for ARMv6 {}", chip.name);
+            debug_sequence = DebugSequence::Arm(Stm32Armv6::create(Stm32Armv6Family::L0));
+        } else if chip.name.starts_with("STM32G0") {
+            tracing::warn!("Using custom sequence for ARMv6 {}", chip.name);
+            debug_sequence = DebugSequence::Arm(Stm32Armv6::create(Stm32Armv6Family::G0));
         } else if chip.name.starts_with("STM32F1")
             || chip.name.starts_with("STM32F2")
+            || chip.name.starts_with("STM32F3")
             || chip.name.starts_with("STM32F4")
             || chip.name.starts_with("STM32F7")
+            || chip.name.starts_with("STM32G4")
+            || chip.name.starts_with("STM32L1")
+            || chip.name.starts_with("STM32L4")
+            || chip.name.starts_with("STM32WB")
+            || chip.name.starts_with("STM32WL")
         {
-            tracing::warn!("Using custom sequence for STM32F1/2/4/7");
-            debug_sequence = DebugSequence::Arm(Stm32fSeries::create());
+            tracing::warn!("Using custom sequence for ARMv7 {}", chip.name);
```

Comment:
maybe we can mention STM32 in this string still? :)

Reply:
The `chip.name` is something like "STM32G431CBUx" so the full log string still says stm32 as well as exactly which chip it is, is that enough?

Reply:
I am an idiot, sorry mate! Nvm my comment :)